### PR TITLE
AT-2106: Added a consistent way to add a label with borders

### PR DESCRIPTION
**Pull request by pavelpichrt**
Adds a label with border, however preserves the existing functionality so that we don't break anything.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -48,12 +48,10 @@ Ads.prototype.init = function(options) {
 			if(validateAdsTrafficResponse) {
 				this.targeting.add(validateAdsTrafficResponse);
 			}
-			return this.initLibrary();
 		})
 		// If anything fails, default to load ads without targeting
-		.catch(e => {
-			return this.initLibrary();
-		});
+		.catch(e => e)
```
Comment (alexflorisca) [inline main.js]: Is this just reformatting? If so, I would leave this as is as it's not related to adding the borders - so just makes it clearer and more relevant if you stick to the scope of the PR.

File: src/scss/base.scss [hunk 2]
```diff
@@ -100,6 +100,36 @@ $ad-max-responsive-width: 1440px;
...
+	min-height 0.35s $o-visual-effects-transition-expand,
+	padding 0.35s $o-visual-effects-transition-expand;
+}
+
+@mixin oAdsLabelWithBorders($pos: 'left') {
+	$padding: 20px;
+
+	font-size: 14px;
+
+	@if($pos == right) {
```
Comment (alexflorisca) [inline src/scss/base.scss]: might need quotes here?

File: main.js [hunk 3]
```diff
@@ -21,17 +21,17 @@ Ads.prototype.init = function(options) {
 	options = options || {};
 	this.config.init();
 	this.config(options);
+	this.config({ nonPersonalized : !options.disableConsentCookie });
```
[the remark this answers is not in the document]
Comment (pavelpichrt) [reply main.js]: done

File: main.js (hunk 1 above)
Comment (pavelpichrt) [reply main.js]: done

File: src/scss/base.scss (hunk 2 above)
Comment (pavelpichrt) [reply src/scss/base.scss]: yep, done

**Pull request merged**